stubs/CoreGenericFunctions.phpstub

```diff
@@ -1482,11 +1482,11 @@ function bin2hex(string $string): string {}
  *
  * @param resource|null $context
  *
- * @return ($associative is 0 ? list<string> : array<string, string|list<string>>)|false
+ * @return ($associative is false ? list<string> : array<string, string|list<string>>)|false
```

Comment:
The change is needed, but the stubs are valid for each php version so this need to stay compatible with old versions.

Can you add handling for both false and 0?

Reply:
This is completed.